### ISSUE TITLE
vscode integration

### ISSUE THREAD

**Issue by awgn** [kowainik/stan#375]
A vscode integration would be very appreciated!

**Comment by chshersh** [comment]
Agree, it would be lovely to have! I'm not involved in the development of the Haskell VSCode plugin, so I don't know what needs to be done to integrate Stan into VSCode. But we're happy to collaborate and provide the required changes to make the integration easier. Maybe it's better to ask HLS maintainers, what is required for Stan integration under the corresponding issue:

* https://github.com/haskell/haskell-language-server/issues/258
